attributes/default.rb

```diff
@@ -66,6 +66,10 @@
     '/etc/datadog-agent'
   end
 
+# The site of the Datadog intake to send Agent data to.
```

Comment:
let's mention here which versions of the Agent actually support this config option